pkg/sink/kafka/admin.go

```diff
@@ -16,170 +16,220 @@ package kafka
 import (
 	"context"
 	"strconv"
-	"strings"
 
-	"github.com/IBM/sarama"
-	"github.com/pingcap/errors"
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
 	"github.com/pingcap/log"
 	"github.com/pingcap/ticdc/pkg/common"
-	cerror "github.com/pingcap/ticdc/pkg/errors"
+	"github.com/pingcap/ticdc/pkg/errors"
 	"go.uber.org/zap"
 )
 
-type saramaAdminClient struct {
-	changefeed common.ChangeFeedID
+type admin struct {
+	ClusterAdminClient
+	client       *kafka.AdminClient
+	changefeedID common.ChangeFeedID
+	timeoutMs    int
```

Comment:
no need to maintain such field, since it's hard code as 1000, just pass it to the method directly.